airbyte-integrations/connectors/source-smartsheets/source_smartsheets/spec.json

```diff
@@ -19,5 +19,13 @@
         "type": "string"
       }
     }
+  },
+  "authSpecification": {
```

Comment:
why do we still use the old `authSpecification` if there are new `advanced_auth` spec?

Reply:
updated now